php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -551,14 +550,11 @@ class NDB_Menu_Filter extends NDB_Menu
      */
     function _setDataTable()
     {
-        // create instance of config object
-        $config =& NDB_Config::singleton();
-        // configure the page pagination
-        $rowsPerPage = $config->getSetting('rowsPerPage');
+        $rowsPerPage = 25;
```

Comment:
Is this still being used? Is there a reason to leave $rowsPerPage in a hard coded state?

Reply:
The class is still being used because DataFrameworkMenu extends it and it's possible that some projects have modules using it so it can't be completely removed yet, but all our menus have been re-done in react with the DataFrameworkMenu.

Reply:
@driusan should we sxplicitely deprecate it? at least to see how many places it's used ?

Reply:
maybe next release? I don't think there's any rush, it's going to take some work to find out how interwoven DataFrameworkMenu is to NDB_Menu_Filter before we can sever that tie and deprecate it. This PR is just removing the (now unused) rowsPerPage config.